nixos/modules/programs/xss-lock.nix

```diff
@@ -12,7 +12,7 @@ in
     lockerCommand = mkOption {
       default = "${pkgs.i3lock}/bin/i3lock";
       example = literalExample ''''${pkgs.i3lock-fancy}/bin/i3lock-fancy'';
-      type = types.string;
+      type = types.str;
```

Comment:
maybe `types.path`? Can this even have cmd args here? @Mic92

Reply:
Yeah I think so, look at the usage of the option a bit later (is called with `-- ${lockerCommand}` in a CLI

Reply:
I guess it should be `separatedString " "` then